events.html

```diff
@@ -155,7 +155,7 @@ <h3 class="event-date-time">
             Building backend services can be challenging, but at Real Dev Squad,
             our members tackle interesting problems regularly.
           </h3>
-          <h3 class="event-date-time">Sunday, April 25th, 12:00pm IST</h3>
+          <h3 class="event-date-time">Sunday, April 25th 2021, 12:00pm IST</h3>
```

Comment:
How you got that this one event is from 2021?

Reply:
I clicked on the register button and saw the event date and year